### ISSUE TITLE
Manual Save of Edited.txt*

### ISSUE THREAD

**Issue by sunjam** [nextcloud/files_texteditor#74]
Auto-save functionality fails occasionally and I'm wondering if a manual save option could implemented, perhaps by clicking on the asterisk (*) symbol displayed by a file name with unsaved changes.

**Comment by jm009** [comment]
By try and error I found out that probably Ctrl+S on the keyboard toggles a save action...

**Comment by Schmuuu** [comment]
Yes, that's correct. CTRL + S triggers a manual save. So that feature is already implemented ;)

**Comment by sunjam** [comment]
On a Mac the key combination is Windows Key + S.  It would be useful if keyboard commands were visible within the Text Editor somewhere.  If that is not do-able would it be possible to implement manual save by clicking *

Might it help to display a pop up if browsing away from the unsaved text, eg. _Unsaved Changes will be discarded. Continue?_
